packages/shared-components/src/pages/dropdown/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html dir="ltr" lang="en">
+<head>
+  <meta charset="utf-8"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0"/>
+  <title>Stencil Component Starter</title>
+
+  <script type="importmap">
+    {
+      "imports": {
+        "@ontotext/workbench-api": "/resources/ontotext-workbench-api.js"
+      }
+    }
+  </script>
+
+  <script type="module" src="/build/shared-components.esm.js"></script>
+  <script nomodule src="/build/shared-components.js"></script>
+  <script src="./main.js" defer></script>
+  <link rel="stylesheet" href="../css/bootstrap.min.css">
+  <link rel="stylesheet" href="../css/font-awesome.min.css">
+</head>
+<body>
+<div style="display: flex; flex-direction: row">
+  <onto-dropdown id="dropdown1" dropdown-button-name="Dropdown 1"
+                 dropdown-button-tooltip="dropdown 1 tooltip"></onto-dropdown>
+
+  <onto-dropdown id="dropdown2" dropdown-button-name="Dropdown 2" dropdown-alignment="0"></onto-dropdown>
```

Comment:
Sorry about this comment (I know it's my fault) but could you change the DropdownItemAlignment to:
```
export enum DropdownItemAlignment {  
  RIGHT = 'right',  
  LEFT = 'left'  
}
```
This way, we can configure the dropdown more meaningfully using `dropdown-alignment="left"` or `dropdown-alignment="right`.

Reply:
changed